openstack/swift/charts/swift_http_importer/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+description: swift-http-import
+name: swift_http_import
```

Comment:
chart name in here (`swift_http_import`) is inconsistent with the directory name (`swift_http_importer`) -- I don't know if this causes any trouble, but it's not desirable at least.